prec_merge_daily.json

```diff
@@ -0,0 +1,155 @@
+{
+    "id": ,
+    "name": "PREC_MERGE",

```

Comment:
"PREC_MERGE" is the same name of the other Collection in this Pull Request, can this one be named PREC_MERGE_DAILY?